src/app.scss

```diff
@@ -38,15 +38,15 @@ body {
 	--main-font: 'Manrope', sans-serif;
 	--mono-font: ui-monospace, SFMono-Regular, SF Mono, Menlo, Consolas, Liberation Mono, monospace;
 	--white: hsl(206, 100%, 94%);
-	--accent-color: hsl(206, 100%, 81%);
-	--accent-color-two: hsl(208, 75%, 82%);
+	--accent-color: var(--primary, hsl(206, 100%, 81%));
+	--accent-color-two: var(--secondary, hsl(208, 75%, 82%));
 	--accent-low-opacity: hsla(205, 91%, 69%, 0.15);
-	--bg-color: hsl(252, 10%, 11%);
+	--bg-color: var(--background, hsl(252, 10%, 11%));
 	--grey-one: hsl(210, 14%, 17%);
 	--grey-two: hsl(212, 19%, 19%);
 	--grey-three: hsl(221, 17%, 26%);
 	--grey-four: hsl(226, 48%, 18%);
-	--grey-five: hsl(208, 30%, 75%);
+	--grey-five: var(--text, hsl(208, 30%, 75%));
 	--grey-six: hsl(230, 9%, 13%);
 	--grey-seven: hsl(240, 9%, 13.5%);
 	--grey-eight: hsla(207, 30%, 75%, 0.577);
```

Comment:
i don't like how this only replaces some colors, it would be better if there was a supplementary hue variable so that things like `--white` don't feel off color

Reply:
custom events can be created